modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/ClusterManagementGroupManager.java

```diff
@@ -828,6 +818,10 @@ public CompletableFuture<Void> onJoinReady() {
         }
     }
 
+    public CompletableFuture<UpdateDistributedConfigurationAction> clusterConfigurationToUpdate() {
+        return updateDistributedConfigurationActionFuture;
```

Comment:
We should complete this future on the node stop. What if somebody's waiting and we stop the node too early?